src/sparseml/pytorch/optim/modifier_lr.py

```diff
@@ -273,6 +273,7 @@ def __init__(
         final_lr: float,
         start_epoch: float,
         end_epoch: float,
+        cycle_epochs: float = -1.0,
```

Comment:
this default value of -1 (also for update_frequency) seems to be malformed based on how the value is used. should it even have a default value?

Reply:
Set default value for cycle_epochs to 1; also verify it's positive if the schedule is cyclic_linear. For update_frequency, it's a dummy value for an unused argument.